packages/http-client-csharp/generator/Microsoft.TypeSpec.Generator.Input/test/InputModelPropertyTests.cs

```diff
@@ -8,143 +8,7 @@ namespace Microsoft.TypeSpec.Generator.Input.Tests
-        public void CanUpdateModelPropertyIsDiscriminator()
-        {
-            var property = InputFactory.Property("TestName", InputPrimitiveType.String, isDiscriminator: false);
-            property.Update(isDiscriminator: true);
-
-            Assert.IsTrue(property.IsDiscriminator);
-        }
-
-        [Test]
-        public void CanUpdateModelPropertySerializedName()
-        {
-            var property = InputFactory.Property("TestName", InputPrimitiveType.String);
-            property.Update(serializedName: "updated_name");
-
-            Assert.AreEqual("updated_name", property.SerializedName);
-        }
-
-        [Test]
-        public void CanUpdateMultiplePropertiesAtOnce()
-        {
-            var property = InputFactory.Property("OriginalName", InputPrimitiveType.String, isRequired: false, isDiscriminator: false);
-            
-            property.Update(
-                name: "UpdatedName",
-                summary: "Updated summary",
-                isRequired: true,
-                isDiscriminator: true
-            );
-
-            Assert.AreEqual("UpdatedName", property.Name);
-            Assert.AreEqual("Updated summary", property.Summary);
-            Assert.IsTrue(property.IsRequired);
-            Assert.IsTrue(property.IsDiscriminator);
-        }
-
-        [Test]
-        public void UpdateWithNullParametersDoesNotChangeProperties()
-        {
-            var property = InputFactory.Property("OriginalName", InputPrimitiveType.String, isRequired: true, isDiscriminator: true);
-            var originalName = property.Name;
-            var originalSummary = property.Summary;
-            var originalDoc = property.Doc;
-            var originalType = property.Type;
-            var originalIsRequired = property.IsRequired;
-            var originalIsReadOnly = property.IsReadOnly;
-            var originalAccess = property.Access;
-            var originalIsDiscriminator = property.IsDiscriminator;
-            var originalSerializedName = property.SerializedName;
-            var originalSerializationOptions = property.SerializationOptions;
-
-            property.Update(); // All parameters are null
-
-            Assert.AreEqual(originalName, property.Name);
-            Assert.AreEqual(originalSummary, property.Summary);
-            Assert.AreEqual(originalDoc, property.Doc);
-            Assert.AreEqual(originalType, property.Type);
-            Assert.AreEqual(originalIsRequired, property.IsRequired);
-            Assert.AreEqual(originalIsReadOnly, property.IsReadOnly);
-            Assert.AreEqual(originalAccess, property.Access);
-            Assert.AreEqual(originalIsDiscriminator, property.IsDiscriminator);
-            Assert.AreEqual(originalSerializedName, property.SerializedName);
-            Assert.AreEqual(originalSerializationOptions, property.SerializationOptions);
-        }
+        // Tests for InputModelProperty have been removed as the Update method was removed
```

Comment:
this file should be removed since it contains nothing

Reply:
There is nothing inherently wrong with calling update on InputModelProperty. I'm not sure what the goal of this PR is.